### PR TITLE
Feature/2592-live-update-tracks-relative-offset-coordinates-on-slider-motion

### PR DESCRIPTION
**Pull request by martinbaerwolff**
OP#2592

### CODE REVIEW

**Comment by martinbaerwolff** [comment]
@briemla as discussed: A popup for user feedback is needed

File: OTAnalytics/plugin_ui/customtkinter_gui/frame_bbox_offset.py [hunk 1]
```diff
@@ -68,3 +72,18 @@ def get_relative_offset_coordintes(self) -> dict:
...
+
+    def _on_slider_change(self, value: Any) -> None:
+        x, y = self._get_slider_values()
+        self.label_x_value.configure(text=round(x, 1))
+        self.label_y_value.configure(text=round(y, 1))
+
+    def _on_slider_release(self, value: Any) -> None:
+        x, y = self._get_slider_values()
+        if self._notify_change is not None:
+            self._notify_change(round(x, 2), round(y, 2))
```
Comment (briemla) [inline OTAnalytics/plugin_ui/customtkinter_gui/frame_bbox_offset.py:84]: Why do we round to a different number of decimal digits?

**Review by briemla — COMMENTED** [review]
Looks good only one question

File: OTAnalytics/plugin_ui/customtkinter_gui/frame_bbox_offset.py (hunk 1 above)
Comment (martinbaerwolff) [reply OTAnalytics/plugin_ui/customtkinter_gui/frame_bbox_offset.py:84]: I changed ot to the same everywhere and also extracted out a constant to the top of the module. It depends on the slider resolution, which I also extracted out to a constant.

**Review by briemla — APPROVED** [review]

**Pull request merged**